industrial_ci/src/isolation/docker.sh

```diff
@@ -15,9 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+export DOCKER_ADDITIONAL_ENVS=${DOCKER_ADDITIONAL_ENVS:-''}
 export DOCKER_COMMIT=${DOCKER_COMMIT:-}
 export DOCKER_COMMIT_MSG=${DOCKER_COMMIT_MSG:-}
 export DOCKER_CREDENTIALS=${DOCKER_CREDENTIALS-.docker .gitconfig .ssh .subversion}
+export DOCKER_ENTRYPOINT=${DOCKER_ENTRYPOINT:-''}
```

Comment:
To keep it in line with the other variables
```suggestion
export DOCKER_ENTRYPOINT=${DOCKER_ENTRYPOINT:-}
```